applications/nrf5340_audio/overlay-dfu.conf

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_MCUBOOT_IMAGE_VERSION="1.0.0+0"
+CONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=y
+
+# Increase system workqueue stack size to prevent stack overflows while handling
+# SMP requests.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_THREAD_NAME=y
+# DFU
+CONFIG_MCUMGR=y
+CONFIG_PCD_APP=y
+CONFIG_FLASH=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_IMG_MGMT=y
+CONFIG_MCUMGR_BUF_SIZE=1024
+
+# BT dfu
```

Comment:
Repace "dfu" to "DFU over BLE" would be great.

Reply:
Revised as suggested.